packages/date-picker/src/_subcomponents/FloatingCalendarWrapper/FloatingCalendarWrapper.tsx

```diff
@@ -2,18 +2,22 @@ import React, { HTMLAttributes, useState } from "react"
 import { OverrideClassName } from "@kaizen/component-base"
 import classnames from "classnames"
 import { usePopper } from "react-popper"
-import { Calendar } from "../Calendar"
-import calendarWrapperStyles from "./CalendarWrapper.module.scss"
+import { Options } from "@popperjs/core"
+import calendarWrapperStyles from "./FloatingCalendarWrapper.module.scss"
 
-interface CalendarWrapperProps
+export interface FloatingCalendarWrapperProps
   extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
-  children: React.ReactElement<typeof Calendar>
+  children: React.ReactNode
   referenceElement: HTMLElement | null
+  popperOptions?: Partial<Options>
 }
 
-export const CalendarWrapper: React.VFC<CalendarWrapperProps> = ({
+export const FloatingCalendarWrapper: React.VFC<
```

Comment:
Since there's no other `CalendarWrapper` it seems odd to call this out as a `Floating` one, perhaps just `CalendarPopper` would suffice since it implements the `usePopper` hook, or `Popover` to be clearer of it's use.